recipes/sphinxjp.themes.solarized/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.1.0" %}
+{% set sha256 = "a9405e7fc875b9d5db80a8615ab7acb2cc6c98310ad65a1a7310652f0df8baaf" %}
+
+package:
+  name: sphinxjp.themes.solarized
+  version: {{ version }}
+
+source:
+  fn: sphinxjp.themes.solarized-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/s/sphinxjp.themes.solarized/sphinxjp.themes.solarized-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - docutils
+    - sphinx
+    - sphinxjp.themecore
```

Comment:
So they only should be need at *build* time if they are in `setup_requires`. Normally it will just be `python` and `setuptools`. Very rarely will something else be included. Could you please double check and update accordingly?